src/optimalcontrol/src/ForwardEuler.cpp

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright (C) 2014,2018 Fondazione Istituto Italiano di Tecnologia
+ * Authors: Francesco Romano, Stefano Dafarra
+ * CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ *
+ * Originally developed for Prioritized Optimal Control (2014)
+ * Refactored in 2018.
+ * Design inspired by
+ * - ACADO toolbox (http://acado.github.io)
+ * - ADRL Control Toolbox (https://adrlab.bitbucket.io/ct/ct_doc/doc/html/index.html)
+ */
+
+#include <iDynTree/Integrators/ForwardEuler.h>
+#include <iDynTree/DynamicalSystem.h>
+#include <iDynTree/Core/Utils.h>
+
+#include <iDynTree/Core/VectorDynSize.h>
+#include <Eigen/Dense>
+#include <iDynTree/Core/EigenHelpers.h>
+
+namespace iDynTree {
+namespace optimalcontrol {
+namespace integrators {
+
+            bool ForwardEuler::allocateBuffers()
```

Comment:
Are this 8 spaces or more?

Reply:
For some reasons ``QTCreator`` does not want me to indent ``namespace`` definitions :sweat_smile: